packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/subscriptions.module.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import '@ovh-ux/ui-kit';
+
+import component from './subscriptions.component';
+import routing from './subscriptions.routing';
+import service from './streams-subscriptions.service';
+
+const moduleName = 'ovhManagerDbaasLogsDetailStreamsSubscriptions';
+
+angular
+  .module(moduleName, [
+    'ngOvhCloudUniverseComponents',
+    'oui',
+    'ovhManagerCore',
+    'ovh-api-services',
```

Comment:
Removed.

Note that this is still widely used in this module:
```
› grep -R "import 'ovh-api-services'" packages/manager/modules/dbaas-logs
packages/manager/modules/dbaas-logs/src/logs/logs.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/encryption-keys.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/home/home.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/home/detail/detail.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/add/add.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/index/index.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/index/add/add.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/dashboards/dashboards.module.js:import 'ovh-api-services';
packages/manager/modules/dbaas-logs/src/logs/detail/dashboards/crud/crud.module.js:import 'ovh-api-services';
```

I'll do another PR to remove it, and to avoid mixing refacto & new feature inside this PR